### PR TITLE
#126485621 ShopOwners Products Page UI

### PR DESCRIPTION
**Pull request by nwochaadim**
#### What does this PR do?

Presents a friendly Products page UI
#### Description of Task to be completed?

Shopowners should see a friendly Products page UI
#### What are the relevant pivotal tracker stories?

126485621

### CODE REVIEW

File: app/assets/javascripts/shop.js [hunk 1]
```diff
@@ -1,3 +1,51 @@
 $(document).ready(function(){
   $('.modal-trigger').leanModal();
-});
\ No newline at end of file
+  $('a.yellow.tooltipped').leanModal();
+  $('input#product_description').characterCounter();
+
+  $('#create_product').click(function(){
+    event.preventDefault();
+    console.log(postData());
```
Comment (andela-tkomolafe) [inline app/assets/javascripts/shop.js]: remove

File: app/assets/javascripts/shop.js [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+function outputErrors(data){
+  html_string = "<ol>";
+  data.forEach(function(item){
+    html_string += "<li>"+item+"</li>";
+  })
+  if ($('#product_image').val()===""){
+    html_string += "<li>Please select a product image</li>";
+  }
+  html_string += "</ol>";
+  console.log(html_string)
```
Comment (andela-tkomolafe) [inline app/assets/javascripts/shop.js]: ditto

File: app/assets/javascripts/shop.js [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+function postData(){
+  data = {};
+  data['name'] = $('#product_name').val();
+  data['description'] = $('#product_description').val();
+  data['quantity'] = $('#product_quantity').val();
+  data['brand'] = $('#product_brand').val();
+  data['size'] = $('#product_size').val();
+  data['price'] = $('#product_price').val();
+  return data;
+}
```
Comment (betkom) [inline app/assets/javascripts/shop.js]: put new line at end of file

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -20,3 +20,4 @@
 @import "single_blog";
 @import "shop_owners";
 @import "shop";
+@import "divider"
```
Comment (andela-tkomolafe) [inline app/assets/stylesheets/application.scss]: ditto

**Comment by andela-tkomolafe** [comment]
@andela-anwocha Please resolve the conflicts, also fix all issues raised by houndci

File: app/assets/stylesheets/divider.css [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+  }
+}
+
+
+/*******************************
+         Site Overrides
+*******************************/
+
```
Comment (tijesunimi-peters) [inline app/assets/stylesheets/divider.css]: Are we using a new CSS framework now?

File: app/controllers/products_controller.rb [hunk 6]
```diff
@@ -12,9 +13,8 @@ def create
     @product = Product.create(product_params)
     @product.update(shop: @shop_owner.shop)
     if @product.errors.empty?
-      redirect_to shop_products_path, notice: MessageService.product_created
+      redirect_to dashboard_path, notice: "Created"
```
Comment (tijesunimi-peters) [inline app/controllers/products_controller.rb]: I feel we can use the MessageService for the message

**Comment by betkom** [comment]
👍

**Pull request merged**